src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -429,6 +429,8 @@ public WFIterator iterator() {
         public QueryIterator(final String chr, final int start, final int end, final List<Block> blocks) throws IOException {
             this.start = start;
             this.end = end;
+            // for error message
+            chrAlias = chr;
```

Comment:
this is already set below and should probably be final

Reply:
I actually need this for the case that an error happens on the first line that is being read....happy to explain more, but this is not a mistake.

Reply:
Oh.  I see.  Maybe write that into the comment?